src/data/wipe-details.json

```
@@ -62,5 +62,9 @@
     {
         "name": "0.15.0.0",
         "start": "2024-08-20T06:00:00.000Z"
+    },
+    {
+        "name": "0.16.0.0",
+        "start": "2024-12-26T00:00:00.000Z"
```

```suggestion
        "start": "2024-12-26T08:00:00.000Z"
```
I think it was 8 in the morning when they wiped and started the update